config/am-scripts/scripts-content/ch-require-mfa-check.js

```diff
@@ -0,0 +1,50 @@
+var fr = JavaImporter(
+    org.forgerock.openam.auth.node.api.Action
+)
+
+var userId = sharedState.get("_id");
+
+var checkMFA = false;
+
+try {
+    // Use AM representation of attribute
+    if (idRepository.getAttribute(userId, "fr-attr-idate1").iterator().hasNext()) {
+        var lastLogin = String(idRepository.getAttribute(userId, "fr-attr-idate1").iterator().next());
+        
+        logger.error("lastLogin: " + lastLogin); // e.g. 20210317114005Z
```

Comment:
very trivial: can we add an appropriate prefix (e.g. '[ MFA-CHECK ]' in all logging instructions? so we can identity in AM logs which script is generating these lines

Reply:
Sure, will do that